docs/docs/dev/security/api-access-control.md

```diff
@@ -23,3 +23,18 @@ Admin resources are `Secret` and `GlobalSecret`.
 
 * `KUMA_ACCESS_STATIC_GENERATE_ZONE_TOKEN_USERS` allows users to generate zone token. Default `mesh-system:admin`.
 * `KUMA_ACCESS_STATIC_GENERATE_ZONE_TOKEN_GROUPS` allows groups to generate zone token. Default `mesh-system:admin`.
+
+## View Envoy config dump
+
+* `KUMA_ACCESS_STATIC_GET_CONFIG_DUMP_USERS` allows users to view Envoy config dump. Default is an empty list.
+* `KUMA_ACCESS_STATIC_GET_CONFIG_DUMP_GROUPS` allows groups to view Envoy config dump. Default: `mesh-system:unauthenticated`, `mesh-system:authenticated`.
```

Comment:
I'm guessing we don't have anything sensitive in config_dump since `mesh-system:unauthenticated` is granted access, right?